omc3/correction/response_madx.py

```diff
@@ -99,9 +99,9 @@ def _generate_madx_jobs(
                 break
             var = variables[var_idx]
             incr_dict[var] = delta_k
-            current_job += f"{var}={var}{delta_k:+f};\n"
+            current_job += f"{var} = {var}{delta_k:+.15f};\n"
             current_job += f"twiss, file='{str(temp_dir / f'twiss.{var}')}';\n"
-            current_job += f"{var}={var}{-delta_k:+f};\n\n"
+            current_job += f"{var} = {var}{-delta_k:+.15f};\n\n"
```

Comment:
why not a more flexible format like e?

Reply:
Actually if we want to be sure, we can use the `#` option which outputs digits untils there are not more.

Reply:
Still needs to decide on a number of decimals to show before the power of 10

Reply:
I propose `{:.15e}`. this should print enough decimals for the given precision.

or we change `delta_k` to a string. this sounds like the most practical solution in this case :p

Reply:
I don't think anyone will ever need that much precision (i.e. there will be no difference in the machine) and probably 4 digits would already be extensive, but as I don't see any disadvantage either ... resolved